src/components/Fonts/FontPreLoader.tsx

```diff
@@ -2,17 +2,15 @@ import React from 'react';
 
 import Head from 'next/head';
 
+import { FONT_CDN } from 'src/utils/fontFaceHelper';
+
 const DEFAULT_LOCALE = 'en';
 
 const LOCALE_PRELOADED_FONTS = {
-  [DEFAULT_LOCALE]: [
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_regular.woff2' },
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_semibold.woff2' },
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_bold.woff2' },
-  ],
-  ar: [{ type: 'font/woff2', location: 'urdu/NotoNaskhArabic-Regular.woff' }],
-  bn: [{ type: 'font/ttf', location: 'NotoSerifBengali/NotoSerifBengali-Regular.ttf' }],
-  ur: [{ type: 'font/woff2', location: 'urdu/NotoNaskhArabic-Regular.woff' }],
+  [DEFAULT_LOCALE]: [{ type: 'font/woff2', location: 'lang/ProximaVara/ProximaVara.woff2' }],
```

Comment:
We were using proxima_vara_semibold and proxima_vara_bold intentionally for non-default `font-weight` values.